server/scripts/setupChainEventListeners.ts

```diff
@@ -198,6 +206,50 @@ const setupChainEventListeners = async (
     });
     return [ node.chain, subscriber ];
   }));
+
+  // Add Erc20 subscribers
+  if (chains === 'all' || chains.includes('erc20')) {
+    const erc20Nodes = await models.ChainNode.findAll({
+      include: [{
+        model: models.Chain,
+        where: { type: 'token' },
+        required: true,
+      }],
+    });
+    const erc20Addresses = erc20Nodes.map((o) => o.address);
+
+    // get ethereum's endpoint URL as most canonical one
+    const ethNode = await models.ChainNode.findOne({
+      where: {
+        chain: 'ethereum'
+      }
+    });
+    const ethUrl = ethNode.url;
+
+    const api = await Erc20Events.createApi(ethUrl, erc20Addresses);
+    // we only need notifications for ERC20s
+    const storageHandler = new EventStorageHandler(models, 'erc20', {});
+    const notificationHandler = new EventNotificationHandler(models, wss);
+    const subscriber = await Erc20Events.subscribeEvents({
+      chain: 'erc20',
+      handlers: [ storageHandler, notificationHandler ],
+      skipCatchup,
+      api,
+      // TODO: discover reconnect range for erc20 will need to provide the specific token's
```

Comment:
?

Reply:
Erc20 doesn't have a `storageFetcher` implemented yet so we can't fetch old events.

Reply:
Old events are fetched through the poller, not the storage fetcher.